homeassistant/components/sutro/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "sutro",
+  "name": "Sutro",
+  "documentation": "https://github.com/ydogandjiev/hass-sutro",
```

Comment:
This should point to the Home Assistant documentation.

Reply:
Thanks! Will work on that soon; still learning how all the pieces come together.